$NetBSD: patch-makefile,v 1.24 2020/10/28 16:56:24 wiz Exp $

Treat all BSDs the same.

Do not set compiler optimization.

--- makefile.orig	2020-10-27 01:00:36.000000000 +0000
+++ makefile
@@ -200,15 +200,15 @@ GENIEOS := bsd
 endif
 ifeq ($(firstword $(filter GNU/kFreeBSD,$(UNAME))),GNU/kFreeBSD)
 OS := freebsd
-GENIEOS := freebsd
+GENIEOS := bsd
 endif
 ifeq ($(firstword $(filter NetBSD,$(UNAME))),NetBSD)
 OS := netbsd
-GENIEOS := freebsd
+GENIEOS := bsd
 endif
 ifeq ($(firstword $(filter OpenBSD,$(UNAME))),OpenBSD)
 OS := openbsd
-GENIEOS := freebsd
+GENIEOS := bsd
 endif
 ifeq ($(firstword $(filter Darwin,$(UNAME))),Darwin)
 OS := macosx
@@ -599,10 +599,6 @@ PROFILER =
 SYMBOLS = 1
 endif
 
-# specify a default optimization level if none explicitly stated
-ifndef OPTIMIZE
-OPTIMIZE = 3
-endif
 
 # set the symbols level
 ifdef SYMBOLS
